diff --git a/include/color_dialog.hpp b/include/color_dialog.hpp
index 5c7653d..895215c 100644
--- a/include/color_dialog.hpp
+++ b/include/color_dialog.hpp
@@ -30,6 +30,8 @@
 
 class QAbstractButton;
 
+using namespace color_widgets;
+
 namespace color_widgets {
 
 class QCP_EXPORT ColorDialog : public QDialog
diff --git a/include/color_list_widget.hpp b/include/color_list_widget.hpp
index 282bea5..7d8e0c5 100644
--- a/include/color_list_widget.hpp
+++ b/include/color_list_widget.hpp
@@ -25,6 +25,8 @@
 #include "abstract_widget_list.hpp"
 #include "color_wheel.hpp"
 
+using namespace color_widgets;
+
 namespace color_widgets {
 
 class QCP_EXPORT ColorListWidget : public AbstractWidgetList
diff --git a/include/color_selector.hpp b/include/color_selector.hpp
index db817d5..48b374d 100644
--- a/include/color_selector.hpp
+++ b/include/color_selector.hpp
@@ -25,6 +25,8 @@
 #include "color_preview.hpp"
 #include "color_wheel.hpp"
 
+using namespace color_widgets;
+
 namespace color_widgets {
 
 /**
